pkg/apis/kops/componentconfig.go

```diff
@@ -384,6 +384,9 @@ type KubeAPIServerConfig struct {
 	KubeletPreferredAddressTypes []string `json:"kubeletPreferredAddressTypes,omitempty" flag:"kubelet-preferred-address-types"`
```

Comment:
Not sure why we would add something to the API, and not wire it up.  The API now has stuff in it that does not work.  Kind of confusing for me.